pmoves/tools/consciousness_harvester.py

```
@@ -0,0 +1,544 @@
+
+    harvester = ConsciousnessHarvester(use_nats=not args.use_http)
+
+    try:
+        if harvester.use_nats:
+            await harvester.connect()
+
+        summary = await harvester.harvest_and_publish(
+            urls, args.output, publish_to_hirag=args.publish
+        )
+
+        print(f"\n[done] Harvest complete:")
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)